pennylane_forest/qpu.py

```diff
@@ -229,6 +202,7 @@ def expval(self, observable):
 
             # Program preparing the state in which to measure observable
             prep_prog = Program()
+            prep_prog += Program(prep_prog.reset())
```

Comment:
This seems like a roundabout way to add a RESET to a program. maybe just add a `RESET` instruction?

Reply:
Thanks for the suggestion, wasn't sure which would be preferred, I'll change it.